sdk/communication/pom.xml

```diff
@@ -31,7 +31,7 @@
         <dependency>
           <groupId>com.azure</groupId>
           <artifactId>azure-communication-chat</artifactId>
-          <version>1.1.3</version> <!-- {x-version-update;com.azure:azure-communication-chat;current} -->
+          <version>1.2.0-beta.1</version> <!-- {x-version-update;com.azure:azure-communication-chat;current} -->
```

Comment:
This updated from running the version update script

Reply:
This may just be a timing thing. Chat also got released recently. You could remove this file from your list and communication PR will handle this or you can check this in, whichever you are comfortable with.